scalalib/src/mill/javalib/sbom/CycloneDX.scala

```diff
@@ -0,0 +1,114 @@
+package mill.javalib.sbom
```

Comment:
Not sure where to place this. Probably in the `contrib` area? 

Especially since I think it will need incubation time to get stable.

Reply:
Moved to contrib.